src/application/machines/devtoolsMachine.tsx

```diff
@@ -23,19 +22,17 @@ export const devtoolsMachine = setup({
   actions: {
     openModal: assign({ modalOpen: true }),
     closeModal: assign({ modalOpen: false }),
-    connectToClient: () => {
+    notifyWaitingForConnection: () => {
       BannerAlert.show({
         type: "loading",
-        content: "Looking for client...",
+        content: "Waiting for client to connect...",
```

Comment:
I updated these messages to be a bit more accurate as to what is happening with the more modern versions of devtools